e2e/suites/actions/upload-download/upload-file.test.ts

```diff
@@ -50,19 +50,73 @@ describe('Upload files', () => {
 
   beforeEach(async (done) => {
     await page.clickPersonalFilesAndWait();
+    await dataTable.doubleClickOnRowByName(folder1);
+    await page.sidenav.openNewMenu();
+    await page.sidenav.menu.uploadFilesInput.sendKeys(`${__dirname}/upload-file.test.ts`);
+    await page.sidenav.closeNewMenu();
+    await page.uploadFilesDialog.uploadDialog.isVisible();
     done();
   });
 
-  afterAll(async (done) => {
+  afterAll(async () => {
     await apis.user.nodes.deleteNodeById(folder1Id);
-    done();
   });
 
   it('Upload a file', async () => {
-    await dataTable.doubleClickOnRowByName(folder1);
-    await page.sidenav.openNewMenu();
-    await page.sidenav.menu.uploadFilesInput.sendKeys(`${__dirname}/upload-file.test.ts`);
+    const uploadedFiles = await dataTable.isItemPresent('upload-file.test.ts');
+    expect(uploadedFiles).toBe(true, 'file not uploaded');
+  });
+
+  it('[T14752064] Close the upload dialog ', async () => {
+    await page.uploadFilesDialog.closeUploadButton.click();
+    await page.uploadFilesDialog.uploadDialog.isPresent();
+    await expect(await page.uploadFilesDialog.uploadDialog.isVisible()).toBe(false);
+  });
+
+  it('[T14752051] Minimize / maximize the upload dialog ', async () => {
+    await page.uploadFilesDialog.minimizeButton.click();
+    await expect(await page.uploadFilesDialog.uploadedFiles.waitNotVisible()).toBe(true);
+
+    await page.uploadFilesDialog.maximizeButton.click();
+    await expect(await page.uploadFilesDialog.uploadedFiles.waitVisible()).toBe(true);
+  });
+
+  describe('[T14752053]', () => {
+    it('Upload history is expunged on browser refresh ', async () => {
+      await page.refresh();
```

Comment:
As I remember the refresh thing is something we want to get rid of in all of E2Es (there is also a PR by @eromano created in alfresco-apps)

Reply:
In the test description it is said that after page refresh the dialog shouldn't be present,  should I delete this e2e test or is there any other way to test it?

Reply:
https://github.com/Alfresco/alfresco-apps/pull/3035#issuecomment-971785029

Yes, please, remove it.

Reply:
Done